packages/core/tests/utils/StorageUtil.test.ts

```diff
@@ -45,8 +45,8 @@ describe('StorageUtil', () => {
     it('should set WalletConnect deep link in localStorage', () => {
       const deepLink = { href: 'https://example.com', name: 'Example Wallet' }
       StorageUtil.setWalletConnectDeepLink(deepLink)
-      const savedDL = SafeLocalStorage.getItem(SafeLocalStorageKeys.DEEPLINK_CHOICE)
-      expect(savedDL).toBe(deepLink.href)
+      const savedDL = SafeLocalStorage.getItem(SafeLocalStorageKeys.DEEPLINK)
+      expect(savedDL).toBe(JSON.stringify({ href: deepLink.href, name: deepLink.name }))
```

Comment:
Since we store primitive value it'll return as pure string instead of an object.